components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -33,6 +33,9 @@ export type DrawerNavGroupProps = SharedStyleProps &
 
         /** Custom element, substitute for title */
         titleContent?: ReactNode;
+
+        /** Divider for the title */
+        titleDivider?: boolean;
```

Comment:
You might want to add this as part of the `SharedStyleProps` instead in `/components/src/core/Drawer/types.tsx`, so that you can specify it at the top level of `<Drawer />`. 

The drawer structure is rather complicated, feel free to have a chat with any one of us.

Reply:
(That `titleColor` prop should also be part of the `SharedStyleProps`; not sure why it is listed here)